velox/exec/HashBuild.cpp

```diff
@@ -812,6 +821,13 @@ void HashBuild::addRuntimeStats() {
           fmt::format("distinctKey{}", i), RuntimeCounter(asDistinct));
     }
   }
+  // Add max spilling level stats if spilling has been triggered.
+  if (spiller_ != nullptr && spiller_->isAnySpilled()) {
```

Comment:
It would be nice to document this new runtime stat in https://facebookincubator.github.io/velox/develop/joins.html#execution-statistics

Reply:
Added. Thanks!